runtimes/spiritnet/src/lib.rs

```diff
@@ -485,7 +497,7 @@ impl pallet_membership::Config<TechnicalMembershipProvider> for Runtime {
 	type WeightInfo = weights::pallet_membership::WeightInfo<Runtime>;
 }
 
-type TipsMembershipProvider = pallet_membership::Instance2;
+type TipsMembershipProvider = pallet_membership::Instance3;
```

Comment:
This would usually be a breaking change, but here I think we are good since we did not implement `MembershipInitialized` and `MembershipChanged`, so no underlying storage changes were made.

Reply:
AFAIK the tippers can vote but the tippers origin has no power. So they can't do much.

Tips are not voted on, but have their separate mechanism.